rpc/src/load_balancer.rs

```diff
@@ -114,13 +114,10 @@ impl LoadBalancer {
                                                 );
 
                                                 {
-                                                    let old_slot = highest_slot.fetch_max(slot.slot, Ordering::Relaxed);
-                                                    if slot.slot > old_slot {
-                                                        if let Err(e) = slot_sender.send(slot.slot)
-                                                        {
-                                                            error!("error sending slot: {e}");
-                                                            break;
-                                                        }
+                                                    if let Err(e) = slot_sender.send(slot.slot)
```

Comment:
this is not going to work super well if RPCs are running behind. note the following graph from our internal + external RPC cluster:
![Screenshot 2024-02-21 at 11 16 42 AM](https://github.com/jito-foundation/jito-relayer/assets/85544055/0fcc942f-373c-408b-997f-1504ee6821c3)